README.md

```diff
@@ -36,8 +36,12 @@ The **docs** component visualizes the collected data with [GitHub Pages](https:/
    git remote add ghe https://git.company.com/scm/hubble
    git push -u ghe master
    ```
-1. Open [`docs/_config.yml`](docs/_config.yml) in your editor and set the `dataURL` that you noted earlier.
-   Commit and push the change to your Hubble repository.
+1. Open [`docs/_config.yml`](docs/_config.yml) in your editor and set the `dataURL` that you noted earlier. Commit and push the change to your Hubble repository:
```

Comment:
Why did you merge the two lines into one? I believe diffs are easier to read with each sentence on a separate line.